apps/email/js/cards/compose.js

```diff
@@ -551,13 +599,13 @@ return [
           makeBubble = true;
           break;
       }
+
       if (makeBubble) {
-        // TODO: Need to match the email with contact name.
-        node.style.width = '0.5rem';
-        var mailbox = model.api.parseMailbox(node.value);
-        this.insertBubble(node, mailbox.name, mailbox.address);
-        node.value = '';
+        entryMatch = this.autocomplete.topMatch ||
```

Comment:
On further reflection, I just removed topMatch. Favoring it in this keyboard character case could actually lead to the user not being able to type in an address if what they wanted to type and send to was a substring match to an existing contact. So if they had amy@a.com in the contacts but wanted to send to amy@a.co, this logic was a problem.

Plus, given that the ; and , are on secondary keyboard screens, the user is more likely to just tap the autocomplete selection than navigate the keyboard to end selection via the keyboard.

I want to add in keyboard support for pressing down/up arrows to select an autocomplete entry, to allow easy selection of autocomplete. I might do that when working on the accessibility ticket for the autocomplete. For that, I'll need to compensate more for blurs occurring but not collapsing the autocomplete list even when there is not a tap triggering a match click. I might need to do that anyway for the screen reader.